crates/re_ui/src/command.rs

```diff
@@ -123,7 +123,7 @@ impl UICommand {
             UICommand::SelectionPrevious => ("Previous selection", "Go to previous selection"),
             UICommand::SelectionNext => ("Next selection", "Go to next selection"),
             UICommand::ToggleCommandPalette => {
-                ("Command palette…", "Toggle the command palette window")
```

Comment:
> Toggle the command palette window

Maybe `Toggle the Command Palette`? Title case Command Palette since it's a concept. Drop the "window" because it's not needed and is overly descriptive.